client/workflows/demos/census-end-to-end.ipynb

```diff
@@ -263,9 +262,9 @@
     "    # log Git information as code version\n",
     "    run.log_code()\n",
     "    \n",
-    "pool = multiprocessing.Pool()\n",
-    "pool.map(run_experiment, hyperparam_sets)\n",
-    "pool.close()"
+    "# NOTE: run_experiment() could also be defined in a module, and executed in parallel\n",
+    "for hyperams in hyperparam_sets:\n",
+    "    run_experiment(hyperams)"
```

Comment:
Bit of a tonguetwister: it's **hyper**-**param**eters
```suggestion
    "for hyperparams in hyperparam_sets:\n",
    "    run_experiment(hyperparams)"
```

Reply:
Fixed.